### PR TITLE
API test for enableApp

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
API test for enableApp
This PR has some skipped scenarios which will pass only after resolving issue #31276

## Related Issue
#31533 

## How Has This Been Tested?
Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Acceptance test

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tests/acceptance/features/apiProvisioning-v1/enableApp.feature [hunk 1]
```diff
@@ -0,0 +1,33 @@
+@api
+Feature: get app info
+As an admin
+I want to be able to enable an disabled app
```
Comment (phil-davis) [inline tests/acceptance/features/apiProvisioning-v1/enableApp.feature]: s/an disabled/a disabled/

File: tests/acceptance/features/apiProvisioning-v1/enableApp.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+So that I can use the app features again
+
+	Background:
+		Given using API version "1"
+
+	Scenario: Admin enables an app
+		Given the app "files" has been disabled
```
Comment (phil-davis) [inline tests/acceptance/features/apiProvisioning-v1/enableApp.feature]: Use comments app for all the tests

**Review by phil-davis — APPROVED** [review]
Make them all about the comments app

**Review by phil-davis — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]
Looks good. This will enable apps again if they got disabled in a scenario.

One day there might be apps that start off disabled, and a scenario enables them. If that ever happens, we can write more AfterScenario code for that.

**Comment by paurakhsharma** [comment]
Yes, we should do that one day.

**Comment by paurakhsharma** [comment]
Backport on #31371

**Pull request merged**